cypress/support/commands.js

```diff
@@ -99,6 +99,42 @@ Cypress.Commands.add('loginUser', ({ email, password }) => {
   );
 });
 
+/**
+ * Visits a URL with the specified locale. Useful for localisation testing.
+ *
+ * @param url {string} The url to visit
+ * @param locale {string} A valid locale to mock
+ */
+Cypress.Commands.add('visitWithLocale', (url, locale) => {
+  cy.visit(url, {
+    onBeforeLoad: (window) => {
+      Object.defineProperty(window.navigator, 'language', { value: locale });
+      Object.defineProperty(window.navigator, 'languages', {
+        value: [locale],
+      });
+    },
```

Comment:
Similarly it might be nice to have a custom command that encapsulates manipulating the `navigator` property like this to test things in different environments